packages/hop-node/src/watchers/SyncWatcher.ts

```diff
@@ -62,9 +62,24 @@ class SyncWatcher extends BaseWatcher {
       this.logger.debug(`syncing from syncFromDate with blockNumber ${this.customStartBlockNumber}`)
     }
 
+    await this.updateBroadcastedStatus()
+
     this.ready = true
   }
 
+  async updateBroadcastedStatus () {
+    const entries = await this.db.transferRoots.getTransferRootsFromTwoWeeks()
+    for (const entry of entries) {
+      if (entry.confirmTxStatus === TxStatus.Broadcasted) {
+        entry.confirmTxStatus = TxStatus.Incomplete
+        if (entry.confirmed) {
```

Comment:
I believe that if `entry.confirmed` is true, then the `entry.confirmTxStatus` would not matter, since it is implied that it is `Completed`. Because of this, I think this line should not exist within the `Broadcasted` check. 

This is a little tricky because we now have two states that represent the same thing (`entry.confirmTxStatus` (when `TxStatus.Brodcasted` is `true`) and `entry.confirmed`). Either way, if the entry has been marked `confirmed`, we know it should 100% be on-chain and completed. Because of this, maybe:

```
      if (!entry.confirmed) {
        if (entry.confirmTxStatus === TxStatus.Broadcasted) {
            entry.confirmTxStatus = TxStatus.Incomplete
        }
    } else {
      entry.confirmTxStatus = TxStatus.complete
    }
```

Open to suggestions here.

Reply:
I think that makes sense but will require a migration (not a big deal though)